framework/src/bundle/BundleRegistry.h

```diff
@@ -23,16 +23,19 @@
 #ifndef CPPMICROSERVICES_BUNDLEREGISTRY_H
 #define CPPMICROSERVICES_BUNDLEREGISTRY_H
 
+#include "cppmicroservices/AnyMap.h"
 #include "cppmicroservices/detail/Threads.h"
-
 #include <condition_variable>
 #include <map>
-#include <unordered_map>
 #include <memory>
 #include <mutex>
 #include <string>
+#include <unordered_map>
+#include <unordered_set>
```

Comment:
This is unnecessary here. Move to BundleRegistry.cpp where it is actually used.

Reply:
Specifically the "#include <unordered_set>," the "#include <unordered_map>" is used in the header file.

Reply:
Done.